examples/nsem/plot_foward_3D.py

```diff
@@ -1,6 +1,11 @@
-# Test script to use SimPEG.MT platform to forward model synthetic data.
+"""
+MT: 3D: Forward
```

Comment:
@grosenkj you will have to fight with a few conflicts no doubt. Hopefully it makes sense what I have moved around.